services/openvpn/session/validator.go

```diff
@@ -34,10 +34,10 @@ type Validator struct {
 }
 
 // NewValidator return Validator instance
-func NewValidator(sessionManager session.Manager, extractor identity.Extractor) *Validator {
+func NewValidator(sessionStorage *session.Storage, extractor identity.Extractor) *Validator {
```

Comment:
Depend on interfaces not on structures

Reply:
Refactored to 2 different interfaces on caller side